packages/fast-components-react-msft/src/action-toggle/action-toggle.spec.tsx

```diff
@@ -0,0 +1,291 @@
+import * as React from "react";
+import * as ShallowRenderer from "react-test-renderer/shallow";
+import * as Adapter from "enzyme-adapter-react-16/build";
+import { configure, mount, shallow } from "enzyme";
+import examples from "./examples.data";
+import MSFTActionToggle, {
+    ActionToggleAppearance,
+    ActionToggleHandledProps,
+    ActionToggleManagedClasses,
+    ActionToggleProps,
+    ActionToggleUnhandledProps,
+} from "./action-toggle";
+import { ActionToggle } from "./index";
+import { ActionToggleClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+
+/*
+ * Configure Enzyme
+ */
+configure({ adapter: new Adapter() });
+
+describe("action toggle", (): void => {
+    const managedClasses: ActionToggleClassNameContract = {
+        actionToggle: "action-toggle",
+        actionToggle_selectedGlyph: "selectedGlyph",
```

Comment:
nit: `action-toggle-selected-glyph` would align with our standards better

Reply:
Not seeing resolution or reasoning for not resolving

Reply:
Updated with changes.